src/go/k8s/tests/e2e/redpanda-schema-registry-sasl/02-retrieve-schema.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: retrive-schema
+spec:
+  template:
+    spec:
+      containers:
+        - name: rpk
+          image: localhost/redpanda:dev
+          env:
+            - name: SCRAM_USERNAME
+              valueFrom:
+                secretKeyRef:
+                  name: schema-registry-schema-registry-sasl
+                  key: username
+            - name: SCRAM_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: schema-registry-schema-registry-sasl
+                  key: password
+          command:
+            - curl
```

Comment:
Unfortunately `curl` does not fail if the status code is not 2xx, you may need to check it manually